### ISSUE TITLE
Colab stopped working with refiner

### ISSUE THREAD

**Issue by sodayo9** [MoonRide303/Fooocus-MRE#108]
I used lllyas's Fooocus for several days on Colab with SDXL refiner and it works . . and I remeber that I once made refiner to work on Colab in FooocusMRE about one month ago but now every time I load a refiner, it says "^C" and the Colab crash... but original Fooocus works with refiner so I wonder it should be a problem

**Comment by barepixels** [comment]
[lllyasviel](https://github.com/lllyasviel) explained why he is not using newer comfyui in his Foooocus here:

https://github.com/lllyasviel/Fooocus/issues/473

**Comment by MoonRide303** [comment]
Yes, I am aware of this problem. Currently MRE supports only base on Colab (no refiner).

**Comment by TimothyAlexisVass** [comment]
How about adding a --colab_friendly argument #120 ?

**Comment by TimothyAlexisVass** [comment]
OK, makes sense
